src/lava/magma/runtime/runtime_service.py

```diff
@@ -158,13 +161,13 @@ def _relay_pm_ack_given_model_id(self, model_id: int):
         ack_relay_port.send(ack_recv_port.recv())
 
     def run(self):
```

Comment:
Please add docstrings and inline explanations.

Reply:
Done